hyperdrive-cli/commands/stakewise/nodeset/upload-deposit-data.go

```diff
@@ -21,3 +21,19 @@ func uploadDepositData(c *cli.Context) error {
 	_, err = swcmdutils.UploadDepositData(sw)
 	return err
 }
+
+func registerNode(c *cli.Context) error {
```

Comment:
This is a new command so it should be in its own file, like `nodeset/register-node.go` instead of living in the upload-deposit-data one.

Reply:
fixed